server/pulp/server/async/worker_utils.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
I can see from the blame log that the history is preserved correctly. Next time I'll check that before suggesting `git mv` is used (which you already did). I looked at the different commits so please go ahead and squash them. git and github both presented the `git mv` as a delete and add so I've reached two conclustion. 1. github is just doing what git does and 2. this an area git could improve on. Thanks for clarifying this.